src/docs/reference/modules/BackgroundTasks/README.md

```diff
@@ -1,3 +1,6 @@
 # Background Tasks (`OrchardCore.BackgroundTasks`)
 
-This module provides tools to manage background tasks. This includes an admin UI to show which background tasks are registered with the ability to enable and disable them. 
\ No newline at end of file
+This module provides tools to manage background tasks. This includes an admin UI to show which background tasks are registered with the ability to enable and disable them. 
+
```

Comment:
Please check how we add video section in other modules

Reply:
I'm not sure about what you mean @hishamco 
I just looked at how we add Youtube videos in our doc and it is pretty much like he did there.

Reply:
https://github.com/OrchardCMS/OrchardCore/blob/532222e1a4d492281ca9dd1990f237d90e8ed382/src/docs/reference/modules/Setup/README.md#video

Reply:
https://docs.orchardcore.net/en/main/docs/reference/modules/Setup/